examples/rust/opensk/src/lib.rs

```diff
@@ -64,25 +67,55 @@ struct WasefireEnv {
     // c: PhantomData<C>,
 }
 
-impl AttestationStore for WasefireEnv {
-    fn get(&mut self, id: &Id) -> Result<Option<Attestation>, Error> {
-        todo!()
+impl Persist for WasefireEnv {
+    fn find(&self, key: usize) -> CtapResult<Option<Vec<u8>>> {
+        let Ok(res) = store::find(key) else {
+            // TODO: What is the correct error code to use here? There doesn't
+            // seem to be an error for value not found or something like that.
+            return Err(CTAP1_ERR_INVALID_COMMAND);
+        };
+        match res {
+            Some(g) => Ok(Some(g.into_vec())),
+            None => Ok(None),
+        }
```

Comment:
nit
```suggestion
        Ok(res.map(|x| x.into_vec()))
```

Alternatively, the whole function could be a single match:

```rust
    Err(_) => Err(Ctap2StatusCode::XXX),
    Ok(None) => Ok(None),
    Ok(Some(data)) => Ok(Some(data.into_vec())),
```

Reply:
Done.